components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -57,15 +56,29 @@ export default function Preferences() {
                 const updatedUser = await getGitpodService().server.getLoggedInUser();
                 setUser(updatedUser);
 
-                toast("Your default workspace timeout was updated.");
+                let toastMessage = <>"Your default workspace timeout was updated."</>;
```

Comment:
**nitpick**: Minor rephrase and dropping the quotes which seem unnecessary.

```suggestion
                let toastMessage = <>"Default workspace timeout has been updated."</>;
```

_UPDATE_: Just saw you also posted something about this in https://github.com/gitpod-io/gitpod/pull/18769#discussion_r1334066581.